src/views/Submit.vue

```diff
@@ -365,6 +380,10 @@ export default {
 		}
 	}
 
+	.submission-message {
+		@include markdown-output;
```

Comment:
```suggestion
		@include markdown-output;
		text-align: center;
```

Without the `text-align: center;` the output looks odd when you have more than one line.

Reply:
@susnux What about this suggestion here?

Reply:
Good point!